File: c11_svg/B_05_Borislav_Mihaylov.rb
def gime_a_circle1 x,y,r
	'<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="'+r.to_s+'" 
stroke="blue" stroke-width="4" fill="red" />'
end
def gime_a_circle2 x,y,r
	'<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="'+r.to_s+'" 
stroke="blue" stroke-width="4" fill="yellow" />'
end
def gime_a_circle3 x,y,r
	'<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="'+r.to_s+'" 
stroke="blue" stroke-width="4" fill="green" />'

end

def rectange x,y,w,h
  '<rect width="'+w.to_s+'" height="'+h.to_s+'" 
style="fill:rgb(0,0,0);stroke-width:3;stroke:rgb(0,0,255)"/>'
end


File.open("result.svg","w") do |f|
	f.write(rectange 00,00,100,250)
	f.write(gime_a_circle1 50,40,40)
	f.write(gime_a_circle2 50,120,40)
	f.write(gime_a_circle3 50,200,40)
	f.write('<text x"30" y="320" font-weight="bold" fill="black">Borislav Mihaylov.text>')	
	f.write('</svg>')
end

File: c21/B_05_Borislav_Mihaylov.js
var array = [0,1,2,3,4,5,6,7,8,9];
var length = array.length;
var i, j;

for (i = 0; i < length; i++) { 
    for(j=0; j< 5; j++){
        fill(150, 100, 200);
        rect(i*20,j*20,20,20);
        fill(255, 255, 255);
        text(""+((i*5)+j),i*20+(10-textWidth(""+((i*5)+j)))/2 + 5,j*20 +15);
    }
}
